src/app/test_executive/zkapps.ml

```diff
@@ -358,26 +359,53 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
       in
       [%log info] "Snapps transaction included in transition frontier"
     in
+    (* Send snapp transactions. *)
     let%bind () =
       section "Send a snapp to create snapp accounts"
         (send_snapp ~logger node parties_create_account)
     in
+    let%bind () =
+      section "Send a snapp to update permissions"
+        (send_snapp ~logger node parties_update_permissions)
```

Comment:
I tried the same but sending these at once wasn't working. for some reason the updates weren't getting applied